utils/python_utils.py

```diff
@@ -14,14 +17,50 @@
         "pytorch_url": "cp311",
     },
 }
+REPO_DIR = Path(__file__).parent.parent
 
 
 def create_conda_env(pyver: str, name: str):
     command = ["conda", "create", "-n", name, "-y", f"python={pyver}"]
     subprocess.check_call(command)
 
 
+def pip_install_requirements(requirements_txt="requirements.txt",
```

Comment:
I'm not sure if it's easy to add a verbose option here and when it's set to true, print out the output of the pip install command below.  When I used `install.py` locally, it installs stuffs silently even with `-v`, which makes it a bit slow to figure out which models install what

Reply:
Adding a verbose option sounds reasonable, but it will be non-trival to turn it on across all models, since each model has its own `install.py`.